apps/dashboard/app/new/page.tsx

```diff
@@ -22,28 +22,8 @@ type Props = {
   };
 };
 
-// Currently unused in this page.
-/* function getBaseUrl() {
-  if (typeof window !== "undefined") {
-    // browser should use relative path
-    return "";
-  }
-
-  if (process.env.VERCEL_URL) {
-    // reference for vercel.com
-    return `https://${process.env.VERCEL_URL}`;
-  }
-
-  // assume localhost
-  return `http://localhost:${process.env.PORT ?? 3000}`;
-} */
-
 export default async function (props: Props) {
-  const user = await auth.getCurrentUser();
-  // make typescript happy
-  if (!user) {
-    return redirect("/auth/sign-in");
-  }
+  const _auth = await getAuth();
```

Comment:
is this needed?

Reply:
It's ensuring we have an authenticated user. We were only asking for a user in order to redirect them if we didn't have an authenticated user. The redirect is built into the getAuth call now, so by calling it we're either going to get an auth return or it's going to redirect. 

We don't need any user data for this page. 

we could do just `await getAuth()` though.

Reply:
gotcha, yeah let's skip the variable in that case 👍

Reply:
cause as it is right now, it looks like someone forgot to remove it
so another engineer might just remove it entirely to clean up